scripts/build-linux-x86_64.sh

```diff
@@ -2,7 +2,7 @@ export SRC_LIB_FILE=libcardano_jni_wrapper.so
 export TARGET_LIB_FILE=linux-x86-64_libcardano_jni_wrapper.so
 export NATIVE_FOLDER=linux-x86-64
 
-cd rust
+cd ../rust
```

Comment:
@edridudi This will break the step in README and CI.  build-linux-*.sh is invoked from root folder.

Reply:
Oh I see.. It didn't work for me when I tried running it locally hence the change.

Reply:
Thanks. No worries. It would be good if you can just rollback files which are not specific to this PR. I will then merge this PR to the branch.